<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Py Smart Mirror</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(0, 0, 0);
QLabel
{
	font-family: HelveticaNeue;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="browserContainer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>684</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="notificationLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>905</y>
      <width>1080</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity: 0;color: #ffffff</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="mainFourFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>1920</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="dateTimeFrame">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>521</width>
       <height>174</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>0</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>August 12 Mon, 2006</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="timeAmPmLabel">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>100</y>
        <width>46</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff</string>
      </property>
      <property name="text">
       <string>AM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff</string>
      </property>
      <property name="text">
       <string>99:99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="timeSecondsLabel">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>64</y>
        <width>46</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff</string>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="quoteFrame">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>1682</y>
       <width>759</width>
       <height>230</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="quoteLabel">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>8</y>
        <width>743</width>
        <height>177</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Quote</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="authorLabel">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>191</y>
        <width>743</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="currentWXFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>174</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="wxTempLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>67Â°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="wxLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Windy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="windLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Also windy.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="currentWxImage">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>3</y>
        <width>80</width>
        <height>55</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="trafficFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>1682</y>
       <width>287</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="trafficArrivalTimeLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Arrival Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="trafficDepartureTimeLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Departure Time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="trafficDestinationLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Destination:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="calendarFrame">
     <property name="geometry">
      <rect>
       <x>781</x>
       <y>190</y>
       <width>290</width>
       <height>420</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="calendarListView">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>43</y>
        <width>274</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>274</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="calendarLabel">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>8</y>
        <width>274</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffffff</string>
      </property>
      <property name="text">
       <string>Calendar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="wxForecastFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>425</width>
       <height>420</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="dayLabel">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>8</y>
        <width>120</width>
        <height>36</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Day</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lowLabel">
      <property name="geometry">
       <rect>
        <x>327</x>
        <y>8</y>
        <width>74</width>
        <height>36</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Low</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="highLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>8</y>
        <width>74</width>
        <height>36</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>High</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="skyImageLabel">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>8</y>
        <width>80</width>
        <height>36</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-pixelsize: 30;
color: #ffffff</string>
      </property>
      <property name="text">
       <string>Sky</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QListWidget" name="wxForecastListView">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>50</y>
        <width>409</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>274</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
